### PR TITLE
docs(issue-templates): add issue templates to repository

### LINKED ISSUE #191: add ISSUE_TEMPLATES to repository

### PR DESCRIPTION
**Pull request by dominguezcelada**
## 📝 Summary

Add issue templates to repository

## ⛱ Motivation and Context

Fix #191 
Relates to https://github.com/octokit/webhooks.js/issues/305

### CODE REVIEW

**Comment by gr2m** [comment]
Are you still working on it? The files are empty

**Comment by dominguezcelada** [comment]
> Are you still working on it? The files are empty

Uops! forgot to commit it. Pushing it now!

**Comment by dominguezcelada** [comment]
@gr2m ready for review now ;)

File: .github/ISSUE_TEMPLATES/04_thanks.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+name: "💝 Thank you"
+about: "@octokit/webhooks.js is awesome 🙌"
```
Comment (gr2m) [inline .github/ISSUE_TEMPLATES/04_thanks.md:3]: The package name is `@octokit/webhooks-definitions`. But I think there are consumers of this repository beyond just npm, so maybe we should call refer to it as just "Octokit Webhooks"?

Same below in lines 9 & 11

**Review by gr2m — APPROVED** [review]

**Pull request merged**